### PR TITLE
Docs 11944 am permissions

### PR DESCRIPTION
**Pull request by sarathecoubian**

### CODE REVIEW

File: modules/ROOT/pages/am-permissions.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
+= Manage Permissions for Anypoint Monitoring
```
[the remark this answers is not in the document]
Comment (kbostrom) [reply modules/ROOT/pages/am-permissions.adoc:1]: I am not sure whether there is more to this topic set, but from what I see in this PR, you could replace the existing Level-1 plus subtopic with a more specific Level-1 narrative:
```suggestion
= Manage Permissions for Anypoint Monitoring
```
See more below:

File: modules/ROOT/pages/am-permissions.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Platform provides predefined permissions and roles that administrators can assign in the *Access Management* section of Anypoint Platform.
+Anypoint Monitoring has two product-specific permissions to control access for different types of users.
+
+== Anypoint Monitoring Permissions
+
+To control access to dashboards, reports, and metrics, use the following permissions for Anypoint Monitoring:
```
Comment (kbostrom) [inline modules/ROOT/pages/am-permissions.adoc:11]:
```suggestion
The *Access Management* section of Anypoint Platform enables you as administrator to user control access to dashboards, reports, and metrics by providing two predefined product-specific permissions:
```

File: modules/ROOT/pages/am-permissions.adoc [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+* Anypoint Monitoring Administrator: +
+View, create, modify, and delete all content in Anypoint Monitoring.
```
Comment (kbostrom) [inline modules/ROOT/pages/am-permissions.adoc:14]:
```suggestion
* Anypoint Monitoring Administrator. Allows users to view, create, modify, and delete all content in Anypoint Monitoring.
```

File: modules/ROOT/pages/am-permissions.adoc [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
++
+* Anypoint Monitoring Viewer: +
+View content in Anypoint Monitoring. Users with this permission cannot modify any Anypoint Monitoring content.
```
Comment (kbostrom) [inline modules/ROOT/pages/am-permissions.adoc:17]:
```suggestion
* Anypoint Monitoring Viewer. Allows users to view but not modify content in Anypoint Monitoring.
```

**Review by kbostrom — DISMISSED** [review]
CE, CU

File: modules/ROOT/nav.adoc [hunk 5]
```diff
@@ -1,5 +1,6 @@
 .xref:index.adoc[Anypoint Monitoring]
 * xref:index.adoc[Overview]
+* xref:am-permissions.adoc[Manage Permissions in Anypoint Monitoring]
```
[the remark this answers is not in the document]
Comment (sarathecoubian) [reply modules/ROOT/nav.adoc:3]:
```suggestion
* xref:am-permissions.adoc[Manage Permissions for Anypoint Monitoring]
```

**Review by emileeburkeVT — APPROVED** [review]

**Pull request merged**